ccc/github.py

```diff
@@ -226,56 +224,11 @@ def log_stack_trace_information_hook(resp, *args, **kwargs):
         ci.util.info(f'Could not log stack trace information: {e}')
 
 
-# TODO: remove this if whitesource is on component descriptor v2
-def github_api_from_component(component: typing.Union[cm.Component, product.model.Component]):
-    if isinstance(component, product.model.Component):
-        github_cfg = github_cfg_for_hostname(host_name=component.github_host())
-        return github_api(github_cfg=github_cfg)
-    elif isinstance(component, cm.Component):
-        source = _get_single_repo(component=component)
-
-        return github_api_from_gh_access(source.access)
-    else:
-        raise ValueError
-
-
-def github_api_from_gh_access(access: cm.GithubAccess):
+def github_api_from_gh_access(
+    access: cm.GithubAccess,
+) -> typing.Union[github3.github.GitHub, github3.github.GitHubEnterprise]:
     if access.type is not cm.AccessType.GITHUB:
         raise ValueError
```

Comment:
nit: I think it would be a good idea to pass `access.type` as argument for `ValueError` (so the wrong/unexpected value is printed as part of the stack-trace)

Reply:
I added the complete access as argument